tests/integration/cassettes/TestLiveThread_test_init.json

```diff
@@ -0,0 +1,164 @@
+{
+  "http_interactions": [
+    {
+      "recorded_at": "2016-12-03T04:01:48",
+      "request": {
+        "body": {
+          "encoding": "utf-8",
+          "string": "grant_type=client_credentials"
+        },
+        "headers": {
+          "Accept": "*/*",
+          "Accept-Encoding": "identity",
+          "Authorization": "Basic <BASIC_AUTH>",
```

Comment:
I've read conftest.py so no issues for me. Anyway, I think [this comment](https://github.com/praw-dev/praw/pull/679#pullrequestreview-10550244) is really useful for contributors.